bin/migration/tables/appaccess.py

```diff
@@ -0,0 +1,72 @@
+from .helpers import check_existing_record, parse_to_timestamp, audit_entry_creation, log_failed_imports
+from datetime import datetime
+import uuid
+
+class AppAccessManager:
+    def __init__(self, source_cursor):
+        self.source_cursor = source_cursor
+        self.failed_imports = set()
+
+    def get_data(self):
+        self.source_cursor.execute("SELECT * FROM public.users WHERE prerole != 'Level 3' OR prerole IS NULL")
+        return self.source_cursor.fetchall()
+
+    def migrate_data(self, destination_cursor, source_data):
+        batch_app_users_data = []
+        id = None
+
+        for user in source_data:
+            user_id = user[0]
```

Comment:
@mazzopardi2  Currently it is diplaying the users who has 'prerole' field value not equal to 'Level3' . But even who has the prerole value as  'Level3' , they have 'Level1' access in the app

Reply:
As discussed this has been adjusted now - thank you :D